spmd/compiler/fusion.py

```diff
@@ -105,10 +154,10 @@ def _scan_graph_for_fusion_elements(
     element_list = []
 
     fe_sequence = [
-        "clone",
+        # "clone",
```

Comment:
why is this commented out?

Reply:
AOT inserts clone nodes in front of the DTensor comm expansion by default (per Sanket it has something to do with assuming people will not be adjusting/using(?) the backwards graph and it's some kind of generic debugging thing).
They are not needed for our use, and it's wasteful memory and perf wise (each clone is a CPU copy of the tensor) and also complicates the graph.  Thus I wanted to eliminate them. 
This is done in this PR https://github.com/pytorch/tau/pull/631 but I haven't finished the review for it. 
For now I commented out  the clone node reference assuming the 631 PR will land shortly and the graph will not have clones...but in case 631 does not land, then wanted to note where we are tracking clone nodes.  (will remove it instead of comment once the clone PR is finalized).